src/vg-tooltip.js

```diff
@@ -667,15 +667,20 @@
           case "string":
         }
       });
+      console.warn("Dropping field(s) " + quanKeys.join(", ") + " from tooltip. Please use vega-lite layers to add quantitative fields to tooltip.")
```

Comment:
No one would understand this warning.  

This is slightly better, but still pretty bad: 
"... from tooltip since we cannot show 
Please use add a layer of non-line/area marks to add tooltip with precise quantitative values."  

Plus, I dont know if people always want this warning.  (Does it always throw warning everytime user mouseover or does it only throw error once.)